CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Please note that this application has changed its name over time.
 It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
+## Kiwix-JS v3.3.0
+
+Released on *TODO*
+
```

Comment:
Maybe you could prepare the line for the detailed changelog too, like in https://github.com/kiwix/kiwix-js/commit/99ec91fb495dfbb1d4f34c1ed8774f938e1d6800#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR11